crates/swc_ecma_transforms_base/src/fixer.rs

```diff
@@ -400,7 +398,11 @@ impl VisitMut for Fixer<'_> {
     fn visit_mut_call_expr(&mut self, node: &mut CallExpr) {
         let old = self.visit_call(&mut node.args, &mut node.callee);
         if let Callee::Expr(e) = &mut node.callee {
-            self.wrap_callee(e)
+            if let Expr::OptChain(_) = &**e {
+                self.wrap(e)
```

Comment:
This is unnecessary -- swc parser and paren_remover won't generate such structure, but it's better to be safe.